.github/workflows/tests_uv.yaml

```
@@ -1,11 +1,11 @@
-name: Tests
+name: Lumigator integration tests
 
 on:
   push:
 
 jobs:
   uv-example:
-    name: python
+    name: Integration tests - Python
```

```suggestion
    name: Backend tests - Python
```
And later do "Backend unit tests" and "Backend integration tests"

---

I think the way that GitHub handles the configuration of required checks (within branch protection rules), means when you change the name of the rule it can't find it anymore and kinda hangs. (Unless there's some clever way I haven't found yet).

So I think it would be better to name this one or the other (unit/integration) and then add the missing one later and extract whatever is needed from this job - meaning the job name doesn't change, and we don't need to manually fiddle the settings to keep things working.

Since we need things up and running for all the current tests to run, I'd like to name this one:
`Backend integration tests - Python`, does that sound OK to you?

---

Fine for me 👍